### PR TITLE
WIP: Schedule adviser workflow

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: thoth/common/workflows.py [hunk 1]
```diff
@@ -317,3 +317,28 @@ def submit_inspection_workflow(
...
+        workflow_parameters = workflow_parameters or {}
+
+        template_parameters["THOTH_ADVISER_JOB_ID"] = adviser_id
+
+        workflow_id: str = self.submit_workflow_from_template(
+            self.openshift.amun_infra_namespace,
+            label_selector="template=adviser",
+            template_parameters=template_parameters,
+            workflow_parameters=workflow_parameters,
+            workflow_namespace=self.openshift.amun_infra_namespace
```
Comment (CermakM) [inline thoth/common/workflows.py]: Should the adviser workflow be run in the Amun Infra namespace?

Comment (goern) [reply thoth/common/workflows.py]: I dont think so
@fridex

Comment (CermakM) [reply thoth/common/workflows.py]: Should be `thoth-backend-stage` iirc.

Comment (fridex) [reply thoth/common/workflows.py]: `thoth-backend-stage`

Comment (pacospace) [reply thoth/common/workflows.py]: Thanks:) I will correct it, I was still testing in test env.

**Pull request merged**